### PR TITLE
Added build step for client library

### PR DESCRIPTION
**Pull request by sedge**
In order to communicate with Makedrive, a client will need access to Filer, rsync and possibly an abstraction layer for communicating with the server. This patch builds the foundation for these requirements.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1,20 +1,48 @@
...
-    }
-  });
+    },
+
+    clean: [ "dist/makedrive.js", "dist/makedrive.min.js" ],
+
+    browserify: {
+      makedriveClient: {
+        src: "./client/src/index.js",
+        dest: "./dist/makedrive.js",
```
Comment (humphd) [inline Gruntfile.js]: I'm not sure if `dist` is the right place for this.  Where does our static express route point?  We'll want to be able to get this file over http.

**Comment by humphd** [comment]
Looks good except for the destination folder.

File: Gruntfile.js (hunk 1 above)
Comment (alicoding) [inline Gruntfile.js]: It needs to be inside `client/`

**Comment by humphd** [comment]
Looks good, r=me

**Pull request closed without merging**